src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -46,6 +46,14 @@
 
 class McpXprCommonTestcase(sas_testcase.SasTestCase):
 
+  def getEmptyCbsdRequestsWithDomainProxies(numberOfElements):
```

Comment:
I think you need to have `self` as the first arg so that you can call as `self.getEmptyCbsdRequestsWithDomainProxies(2)` later.

When you make that change, can you also change to `number_of_elements` so that the var naming style is more consistent?